tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -2743,11 +2756,29 @@ public function userExpiresTheLastShareOfResourceInsideOfTheSpace(
 			if ($response->getStatusCode() === 200) {
 				$this->featureContext->shareNgAddToCreatedUserGroupShares($response, $resource, $spaceName);
 			}
-			$this->featureContext->setResponse($response);
-		} else {
-			$rows['permissions'] = (string)$this->featureContext->getLastCreatedUserGroupShare()->permissions;
-			$this->featureContext->setResponse($this->updateSharedResource($user, $rows));
+			return $response;
 		}
+		$rows['permissions'] = (string)$this->featureContext->getLastCreatedUserGroupShare()->permissions;
+		return $this->updateSharedResource($user, $rows);
+	}
+
+	/**
+	 * @Given user :user has expired the last share of resource :resource inside of the space :spaceName
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 * @throws GuzzleException|JsonException
+	 */
+	public function userHasExpiredTheLastShareOfResourceInsideOfTheSpace(
+		string $user,
+		string $resource,
+		string $spaceName,
+	): void {
+		$response = $this->expireResourceShare($user, $resource, $spaceName);
+		$this->featureContext->setResponse($response);
```

Comment:
Given step should not save the response; instead, check here that the request is successful